cloud_sql/mysql_instance_switchover/dr_replica_create/main.tf

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+# [START cloud_sql_mysql_instance_switchover_dr_replica_create]
+resource "google_sql_database_instance" "original-primary" {
+  name = "mysql-original-primary-instance"
+  # You can specify any region.
```

Comment:
There are some really important comments in these samples, but I think that the ones for `region` and `# Optional. Add more settings.` are superfluous and should be removed.

Reply:
thanks!

Reply:
Removed those comments